editor/.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 /.env
 /.env.*
 /.idea
+.yarn/
```

Comment:
@pyshx @soneda-yuya I took it from -> https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

<img width="1008" alt="image" src="https://github.com/user-attachments/assets/b529a707-1962-4611-ba67-29721ecedec8" />